examples/Clustering.ipynb

```diff
@@ -21,7 +21,7 @@
        "(1000, 1536)"
       ]
      },
-     "execution_count": 1,
```

Comment:
Let's re-run from scratch and reset all of these execution counts, but asides from that, all good!